html/elements/link.json

```diff
@@ -793,7 +793,8 @@
                   "version_added": "≤79"
                 },
                 "firefox": {
-                  "version_added": "3"
+                  "version_added": "3",
+                  "notes": "Before Firefox 127, only HTTP pages were supported."
```

Comment:
I wasn't sure if this warrants a `partial_implementation` before 127.

Reply:
I would say this warrants `partial_implementation` and an array support statement given how important HTTPs has been in the last 10+ years.

Reply:
Fixed via aee4c3f70.